packages/ts-morph/src/compiler/types/Type.ts

```diff
@@ -233,20 +233,31 @@ export class Type<TType extends ts.Type = ts.Type> {
     }
 
     /**
-     * Gets the target type of a type reference if it exists.
+     * Returns the generic type when the type is a type reference, returns itself when it's
+     * already a generic type, or otherwise returns undefined.
```

Comment:
```suggestion
     * Gets the generic type when the type is a type reference, itself when it's
     * already a generic type, or otherwise returns undefined.
```

Better?